AngularLanguageService/AngularLanguageClient.cs

```diff
@@ -38,7 +38,6 @@ public class AngularLanguageClient : ILanguageClient, ILanguageClientCustomMessa
         public IEnumerable<string> FilesToWatch => null;
 
         public event AsyncEventHandler<EventArgs> StartAsync;
-        public event AsyncEventHandler<EventArgs> StopAsync;
```

Comment:
This shouldn't build successfully - `StopAsync` is part of `ILanguageClient`.
So unfortunately, it's needed.

Reply:
Oops! This was a hasty fix. I will abandon this and handle the warnings a different way

Reply:
Oops, this was a hasty fix. I'll handle the warning a different way.